docs/python/py_sdk_docs.md

```diff
@@ -71,7 +71,8 @@ Once you have configured the SDK, it's super easy to use! Simply use the CodeLig
 
 ### Using the Error Catcher Decorator
 Each decorator only applies to the one function defined directly below it. In the decorator, specify the email address of the user in your organization who should receive the notification. 
-
+Note that for frameworks like Flask that use decorators to denote a route, the Codelighthouse `error_catcher` decorator 
+**must** be the innermost decorator!
```

Comment:
This technically isn't true -- the reason that flask requires CLH to be inside of it is because Flask's decorator sets up the context for what happens when the function is called after app.run() has been called.  It's a symptom of how WSGI manages processes.

This isn't a problem that will happen with 99.9% of decorators.  Technically we want it to actually be the outer-most decorator since that way it will also capture issues that the more inner decorators will throw.

Reply:
@willtraweek what should I do about it? hard to know which ones we do vs don't want to come before

Reply:
Not necessarily -- the only ones that need to be outside of the codelighthouse one are decorators that change what triggers the function.  Django, Flask, and Pyramid are the only ones I can think of ATM.

As for direct changes, we shouldn't say it **must** be the innermost decorator.  Rather, I would rephrase the above to the below:
```
Note that the CodeLighthouse decorator must be inside of decorators for web frameworks (Flask, Django, Pyramid)
```